core/trino-main/src/test/java/io/trino/operator/join/unspilled/TestHashJoinOperator.java

```diff
@@ -191,16 +191,16 @@ public void testInnerJoin(boolean parallelBuild, boolean probeHashEnabled, boole
         assertOperatorEquals(joinOperatorFactory, taskContext.addPipelineContext(0, true, true, false).addDriverContext(), probeInput, expected, true, getHashChannels(probePages, buildPages));
     }
 
-    @Test(dataProvider = "singleBigintLookupSourceProvider")
```

Comment:
Let's keep it please. This way we test both `PagesHash` implementations with same tests.
With this commit you cut significant test coverage

Reply:
Removed the commit removing these tests